### DISCUSSION TITLE
有关验证码无法显示的问题

### DISCUSSION THREAD

**Discussion by Yakkai-FMA** [mikumifa/biliTickerBuy#144] [提问专区]
已完成登录后
在“训练你的验证码速度”处选择”手动“过验证码，返回了“gt”与“challenge”并有“重新生成”和“完成验证码后点此此按钮”但不弹出显示手动过验证码的界面
![image](https://github.com/mikumifa/biliTickerBuy/assets/130232080/3088eccd-8243-4a00-8d91-562de87d71f2)

**Comment by mikumifa** [comment]
看一下是不是忘记关梯子了

**Reply by Yakkai-FMA** [reply]
使用梯子时能够返回gt和challenge，关闭梯子后我这里好像无法生成验证码？不返回gt和challenge并显示在加载。请问这个情况对实际抢票是否会产生影响？

**Comment by mikumifa** [comment]
抢票时候不要开梯子

**Comment by mikumifa** [comment]
你在仔细检查一下你得网络环境，我觉得你可能网络环境有问题，你以为你没开梯子，但是你开了

你以为你关梯子了，实际上还是开的

把握不住就重启一下

**Comment by Yakkai-FMA** [comment]
我已经尝试了重启，并关闭梯子。现在可以返回gt与challenge但依然无法显示验证码界面。在我的两台设备上出现了同样的情况

**Comment by mikumifa** [comment]
我这边可以啊。。

**Comment by mikumifa** [comment]
你开一下f12，然后启动一下验证码，看看控制台有什么报错

**Comment by Yakkai-FMA** [comment]
![Uploading image.png…]()

**Reply by mikumifa** [reply]
图片没传上去

**Reply by mikumifa** [reply]
![image](https://github.com/mikumifa/biliTickerBuy/assets/99951454/e4e3f161-474f-484f-ac84-26a31c924df2)
我估计你是这个原因

**Reply by mikumifa** [reply]
我里面使用百度的源，但是百度是不能外网访问的

你可以看看你当前网络能不能访问

https://libs.baidu.com/jquery/1.10.2/jquery.min.js

**Comment by Yakkai-FMA** [comment]
确实是这个问题，我去看看有什么解决办法。

**Comment by mikumifa** [comment]
下一个版本就解决了

**Comment by mikumifa** [comment]
现在先测本地自动过码，那个用的多

**Comment by mikumifa** [comment]
等一下 v2.8.4-patch.2 版本

**Comment by Yakkai-FMA** [comment]
好的，感谢本地没有问题
